cluster-scope/base/operators.coreos.com/catalogsources/rapidast/catalogsource.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: rapidast-catalog
+  namespace: rapidast
```

Comment:
@jpweiser this namespace will also need to be added, you will need to add it to `base` much like the other manifests and then pulled in to the smaug overlay: 

namespace manifests are found here: 
https://github.com/operate-first/apps/tree/master/cluster-scope/base/core/namespaces

you will need [to set a quota for the namespace](https://www.operate-first.cloud/apps/content/cluster-scope/add_resource_quotas.html).

Reply:
This namespace was added previously https://github.com/operate-first/apps/tree/master/cluster-scope/base/core/namespaces/rapidast

I believe the quota is set [here](https://github.com/operate-first/apps/blob/master/cluster-scope/base/core/namespaces/rapidast/kustomization.yaml#L8)

Reply:
@HumairAK Does the above comment address your concerns, or did I miss something?